ORST/Assets/Scripts/Core/Movement/TeleportAudioHandler.cs

```diff
@@ -0,0 +1,72 @@
+using UnityEngine;
+
+namespace ORST.Core {
+    [System.Serializable]
+    struct AudioClipInfo {
```

Comment:
Add an explicit accessibility modifier like

```suggestion
    private struct AudioClipInfo {
```
or
```suggestion
    internal struct AudioClipInfo {
```